xml/templates/message_templates/event_online_receipt_html.tpl

```diff
@@ -31,7 +31,12 @@
      <p>{$event.confirm_email_text|htmlize}</p>
 
     {else}
-  <p>Thank you for your participation.  This letter is a confirmation that your registration has been received and your status has been updated to <strong>{if $participant_status}{$participant_status}{else}{if $isOnWaitlist}waitlisted{else}registered{/if}{/if}</strong>.</p>
+  <p>{if $participant_status}
+  {ts}Thank you for your participation.  This letter is a confirmation that your registration has been received and your status has been updated to {$participant_status}.{/ts}
```

Comment:
As pointed out by Coleman in #7599 using variable inside ts tag should avoided. 
Also, you need to handle the upgrade for the changes made to message templates.